views/includes/list/update-review-bo-mo.html

```diff
@@ -0,0 +1,78 @@
+<div class="govuk-summary-card">
+    <div class="govuk-summary-card__title-wrapper">
+        <h2 class="govuk-summary-card__title">What you have reviewed</h2>
+    </div>
+    <div class="govuk-summary-card__content">
+        <dl class="govuk-summary-list">
+            {% set savedBoIndividual = beneficial_owners_individual | rejectattr("ch_reference", "undefined") %}
```

Comment:
no need for set - just do it in for

Reply:
We thoroughly tested this, but the page is throwing an internal service error whenever we try to use the "rejectattr" method, the error is:

` {"created":"2023-05-04T10:13:55.592+00:00","event":"error","namespace":"overseas-entities-web","data":{"message":"An error has occurred. Re-routing to the error screen - Template render error: (/app/views/update/update-beneficial-owner-type.html)\n  Template render error: (/app/views/includes/list/update-review-bo-mo.html)\n  TypeError: Cannot read properties of undefined (reading 'filter')\n`

In the interest in time, we've reverted to what was there previously, which worked.